challenge-one/src/constants.ts

```diff
@@ -0,0 +1,10 @@
+export const METHODS = [
```

Comment:
```suggestion
export const CREATE_UPDATE_AGENT_METHODS = [
```
Suggestion for a more descriptive name.